### DISCUSSION TITLE
Hello World - 夏沫花火zzz🌙 (Muska_Ami)のLife

### DISCUSSION THREAD

**Discussion by Muska-Ami** [Muska-Ami/muska-ami-blog#65] [General]
https://blog.1l1.icu/2023/09/24/hello-world/? 

 你好世界世界你好，你好世界。由于种种原因，Blog进行一次大更新，友链等信息后续会迁移。文章数据只保留了几篇认为值得放出来的，其他的都会放在后续的存档站 Endlife 。 Welcome to Hexo ! This is your very first post. Check documentation  for more info. If you get any problems when

**Comment by Muska-Ami** [comment]
测试评论

**Comment by chenziqian0625** [comment]
没人水我来水了(
